src/ui/layout/index.md

```diff
@@ -188,6 +188,71 @@ class MyApp extends StatelessWidget {
   or you can build your own set of custom widgets.
 {{site.alert.end}}
 
+#### Cupertino apps
+
+To create a `Cupertino` app, use a [`CupertinoPageScaffold`][] widget.
```

Comment:
Since the reader may not be familiar with `CupertinoApp` at this point, maybe expand this to say "`CupertinoApp` and `CupertinoPageScaffold`.